src/components/StudentTest.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState }from "react";
+import { useSelector, useDispatch} from 'react-redux';
+import styles from './StudentTestStyle'
+import { Link } from "react-router-dom";
+import speaker from '../assets/speakerToo.png'
+//import { addMyWords, removeMyWords, addMasteredWords, removeMasteredWords } from '../redux/slices/words';
+
+const Test = (()=>{
+    const dispatch = useDispatch();//dispatch allows you to use reducers written in a slice
+    const myWords = useSelector(state => state.word.myWords);//useSelector() allows access to state variables
+    const [currentSpelling, changeSpelling] = useState("");
+    const [testQuestion, incrementQuestion] = useState(0);
+    const [testResults, addResult] = useState([]);
+    const hearWord = () => {
+        //get audio file for the word the user is currently on and play it.
+
+        const wordAudio = require("../assets/"+ myWords[testQuestion]+".mp3");
+        new Audio(wordAudio).play();
+    }
+    const handleNextQuestion = () => {
+        incrementQuestion(testQuestion+1);
+        addResult([...testResults,currentSpelling]);
+        changeSpelling("");
+        
+    }
+    return (
+        testQuestion<myWords.length?
```

Comment:
<>
  { testQuestiong < myWords.length ? (
    ... some UI to render
  ) : (
    .. other UI to render
  )